### PR TITLE
feature/gammatemplate carlos-calvo

### PR DESCRIPTION
**Pull request by carlosclatg**

### CODE REVIEW

File: staff/carlos-calvo/feature-gammatemplate/about.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=PT Sans">
+    <link rel="stylesheet" href="./styles.css">
+    <title>Document</title>
```
Comment (martimalek) [inline staff/carlos-calvo/feature-gammatemplate/about.html]: You could put a title here, maybe something like "About Gamma".

File: staff/carlos-calvo/feature-gammatemplate/contact.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        </div>
+    </header>
+    <div class = "subheader">
+        <h2 class ="subheader__title">CONTACT US</h2>
+    </div>
+    <section class="sectionmain">
+        <div class="sectionmain__formulario">
+            <form action="">
```
Comment (martimalek) [inline staff/carlos-calvo/feature-gammatemplate/contact.html]: The attribute action is not being used so you could delete it.

File: staff/carlos-calvo/feature-gammatemplate/contact.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+                Name:<br>
```
Comment (martimalek) [inline staff/carlos-calvo/feature-gammatemplate/contact.html:31]: Maybe you could use some heading tags here.

File: staff/carlos-calvo/feature-gammatemplate/contact.html [hunk 4]
```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=PT Sans">
+    <link rel="stylesheet" href="./stylescontact.css">
+    <title>Document</title>
```
Comment (martimalek) [inline staff/carlos-calvo/feature-gammatemplate/contact.html]: What about "Contact Gamma"?

File: staff/carlos-calvo/feature-gammatemplate/about.html (hunk 1 above)
Comment (carlosclatg) [reply staff/carlos-calvo/feature-gammatemplate/about.html]: Corrected

File: staff/carlos-calvo/feature-gammatemplate/contact.html (hunk 3 above)
Comment (carlosclatg) [reply staff/carlos-calvo/feature-gammatemplate/contact.html:31]: Ok

**Pull request merged**